EmpowerPlant/EmpowerPlantViewController.swift

```diff
@@ -8,21 +8,149 @@
 import UIKit
 import Sentry
 
-class EmpowerPlantViewController: UIViewController {
+class EmpowerPlantViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
 
+    let context = (UIApplication.shared.delegate as! AppDelegate).persistentContainer.viewContext
+    
+    let tableView: UITableView = {
+        let table = UITableView()
+        table.register(UITableViewCell.self, forCellReuseIdentifier: "cell")
+        return table
+    }()
+    
+    var products = [Product]()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         title = "Empower Plant"
-        // Do any additional setup after loading the view.
+
+        self.view.addSubview(tableView)
+        tableView.delegate = self
+        tableView.dataSource = self
+        
+        // Dev Note - Comment this out and to see the green background and no data in the rows
+        tableView.frame = view.bounds
+        
         configureNavigationItems()
+
+        // TODO - clear Db on startup, and insert the objects from server into the db
+        getAllProductsFromDb()
+        getAllProductsFromServer()
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return products.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let model = products[indexPath.row]
+        let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath)
+        cell.textLabel?.text = model.title
+        return cell
+    }
+    
+    func getAllProductsFromServer() {
+        print("getAllProductsFromServer")
+        let url = URL(string: "https://application-monitoring-flask-dot-sales-engineering-sf.appspot.com/products-join")!
+        var request = URLRequest(url: url)
+        request.setValue("application/json", forHTTPHeaderField: "Content-Type")
+        
+        struct Product1: Decodable {
```

Comment:
Need to extend the existing `Product` class instead so that it's Decodable. This involves re-generating the class files using XCode. That will go on the next. PR @cstavitsky